spec/converter_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'spec_helper'
+require 'ronin/masscan/converter'
+require 'masscan/output_file'
+require 'tempfile'
+
+RSpec.describe Ronin::Masscan::Converter do
+  let(:fixtures_path) { File.expand_path(File.join(__dir__, '..', 'spec', 'fixtures')) }
+  let(:masscan_path)  { File.join(fixtures_path, 'converter', 'masscan.json') }
+  let(:masscan_file)  { Masscan::OutputFile.new(masscan_path) }
+  let(:expected_json) do
+    "[{\"status\":\"open\",\"protocol\":\"tcp\",\"port\":80,\"reason\":[\"syn\",\"ack\"],\"ttl\":54,\"ip\":\"93.184.216.34\",\"timestamp\":\"2021-08-26 08:50:21 +0000\"}]"
+  end
+
+  describe '.convert_file' do
+    it 'must convert masscan file and wirte it into a file' do
```

Comment:
Should I add more examples?

Reply:
You could add specs for when `input_format:` is given, or when `format:` is explicitly given.

Reply:
If we had a `spec/fixtures/converters/input.csv` file, we could then test `input_format: :csv, format: :json`.